game/campaignloader/campaignairwingconfig.py

```diff
@@ -71,11 +71,27 @@ def from_campaign_data(
         cls, data: dict[Union[str, int], Any], theater: ConflictTheater
     ) -> CampaignAirWingConfig:
         by_location: dict[ControlPoint, list[SquadronConfig]] = defaultdict(list)
+        carriers = theater.find_carriers()
+        lhas = theater.find_lhas()
         for base_id, squadron_configs in data.items():
+            base: Optional[ControlPoint] = None
             if isinstance(base_id, int):
                 base = theater.find_control_point_by_airport_id(base_id)
             else:
-                base = theater.control_point_named(base_id)
+                try:
+                    base = theater.control_point_named(base_id)
+                except:
+                    if base_id == "Red CV":
+                        base = next((c for c in carriers if not c.captured), None)
+                    elif base_id == "Blue CV":
+                        base = next((c for c in carriers if c.captured), None)
+                    elif base_id == "Red LHA":
+                        base = next((l for l in lhas if not l.captured), None)
+                    elif base_id == "Blue LHA":
+                        base = next((l for l in lhas if l.captured), None)
+
+            if base is None:
+                raise ValueError(f"No valid control points found {base_id}")
```

Comment:
Perhaps log an error and skip the base? Raising an exception seems a little destructive since it would mean the air-wing won't get loaded up at all...